server/graphql/schema/queries/getHomeViewer.ts

```diff
@@ -21,6 +22,13 @@ export const getHomeViewer = {
         where: { id: userId },
       });
     }
-    return { user };
+
+    const announcements = await Announcement.findAll({
+      where: { showOnHomepage: true },
+      order: [['createdAt', 'DESC']],
+      limit: 5,
```

Comment:
good thing to limit these but if we are going to limit it we should put text on admin panel side that only 5 recent posts will be shown on home page (and we may also want to put handler on create query to check if already have 5 'showOnHomepage' before creating one and display proper error message).

add it to here https://trello.com/c/O1Qywisc/33-tech-debt-better-error-and-loading-check